.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: Release
+
+on:
+  release:
+    types: [ "published" ]  
+    
+permissions:
+  checks: write
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+          fetch-depth: 0
+
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v4
+
+    - name: Build
+      run: ./build.sh build
+      shell: bash
+
+    - name: Pack
+      run: dotnet pack -c Release -o nuget --no-build --nologo
+      shell: bash
+
+    - name: Publish NuGet package
+      env:
+        NUGET_API_KEY: ${{ secrets.NUGET_API_KEY }}
+      run: dotnet nuget push nuget/*.nupkg --api-key $NUGET_API_KEY --source https://api.nuget.org/v3/index.json
```

Comment:
I guess API key is already presented

Reply:
Should we try to do a release? Assuming the secret is set?